src/containers/Tenant/i18n/en.json

```diff
@@ -36,6 +36,7 @@
   "actions.dropTopic": "Drop topic...",
   "actions.dropView": "Drop view...",
   "actions.alterTable": "Alter table...",
+  "actions.addTableIndex": "Add index...",
```

Comment:
Don't we want to add action "Drop index..." directly to index entity?

Reply:
added INDEX_SET